test-ci.js

```diff
@@ -87,6 +87,7 @@ let csvNA = null;
     'Windows-WebGL',
     'Linux-WASM',
     'Linux-WebGL',
+    'Mac-fast',
```

Comment:
please update from Line#82 to Line#90, by referring to https://github.com/intel/webml-polyfill/blob/master/test/tools/RegressionChecker/src/main.js#L89